src/DataProtection/custom/Helpers/BackupcenterHelper.ps1

```diff
@@ -30,6 +30,79 @@
 	}
 }
 
+function GetResourceGroupIdFromArmId {
+	[Microsoft.Azure.PowerShell.Cmdlets.DataProtection.DoNotExportAttribute()]
+	param(
+		[Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [System.String]
+        $DatasourceId
+	)
+	
+	process{
+
+        $ResourceArray = $DataSourceId.Split("/")
+        $ResourceRG = "/subscriptions/" + $ResourceArray[2] + "/resourceGroups/" + $ResourceArray[4]
+
+		return $ResourceRG
+	}
+}
+
+function GetSubscriptionNameFromArmId {
+	[Microsoft.Azure.PowerShell.Cmdlets.DataProtection.DoNotExportAttribute()]
+	param(
+		[Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [System.String]
+        $DatasourceId
+	)
+	
+	process{
+
+        $ResourceArray = $DataSourceId.Split("/")
+        $SubscriptionName = "/subscriptions/" + $ResourceArray[2]
+        
+		return $SubscriptionName
+	}
+}
+
+
+function GetResourceNameFromArmId {
+	[Microsoft.Azure.PowerShell.Cmdlets.DataProtection.DoNotExportAttribute()]
+	param(
+		[Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [System.String]
+        $DatasourceId
```

Comment:
Instead of datesource Id - can we say just Id ?

Reply:
Aditya's Document says the parameter name should be DatasourceId.So should I still go ahead and change it?